### ISSUE TITLE
tf.keras.model layer shows nothing

### ISSUE THREAD

**Issue by cherytony** [tensorflow/tensorflow#21352]
tf.keras.Model.layers shows nothing?

![2018-08-03_115300](https://user-images.githubusercontent.com/26696338/43623464-e6dc594e-9713-11e8-8711-c403f6b20995.gif)

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
Mobile device

**Comment by tensorflowbutler** [comment]
It has been 15 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 30 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
We are closing this issue for now due to lack of activity. Please comment if this is still an issue for you. Thanks!
